ipam/allocator.go

```diff
@@ -603,6 +612,7 @@ func (alloc *Allocator) establishRing() {
 	}
 
 	alloc.awaitingConsensus = true
+	alloc.paxos.SetQuorum(alloc.getQuorum())
```

Comment:
We do: `determineQuorum` has `Log.Println("Assuming quorum size of", quorum)`

It won't be printed if it's been set by `--init-peer-count`